site2/docs/administration-isolation.md

```diff
@@ -0,0 +1,84 @@
+---
+id: administration-isolation
+title: Pulsar isolation
+sidebar_label: Pulsar isolation
+---
+
+In an organization, a Pulsar instance provides services to multiple teams. When organizing the resources across multiple teams, you want to make a suitable isolation plan to avoid the resource competition between different teams and applications and provide high-quality messaging service. In this case, you need to take resource isolation into consideration and weigh your intended actions against expected and unexpected consequences.
+
+To enforce resource isolation, you can use the Pulsar isolation policy, which allows you to allocate resources (**broker** and **bookie**) for the namespace.
+
+## Broker isolation
+
+In Pulsar, when namespaces (more specifically, namespace bundles) are assigned dynamically to brokers, the namespace isolation policy limits the set of brokers that can be used for assignment. Before topics are assigned to brokers, you can set the namespace isolation policy with a primary or a secondary regex to select desired brokers.
+
+You can set a namespace isolation policy for a cluster using one of the following methods. 
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Admin CLI-->
+
+```
+pulsar-admin ns-isolation-policy set options
+```
+
+For more information about the command and descriptions, see [here](http://pulsar.apache.org/tools/pulsar-admin/2.8.0-SNAPSHOT/#-em-set-em-).
+
+**Example**
+
+```shell
+bin/pulsar-admin ns-isolation-policy set \
+--auto-failover-policy-type min_available \
+--auto-failover-policy-params min_limit=1,usage_threshold=80 \
+--namespaces my-tenant/my-namespace \
+--primary 10.193.216.*  my-cluster policy-name
+```
+
+<!--REST API-->
+
+[PUT /admin/v2/namespaces/{tenant}/{namespace}](https://pulsar.apache.org/admin-rest-api/?version=master&apiversion=v2#operation/createNamespace)
+
+<!--Java admin API-->
+
+For how to set namespace isolation policy using Java admin API, see [here](https://github.com/apache/pulsar/blob/master/pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/NamespacesImpl.java#L251).
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Bookie isolation
+
+A namespace can be isolated into user-defined groups of bookies, which guarantees all the data that belongs to the namespace is stored in desired bookies. The bookie affinity group uses the BookKeeper [rack-aware placement policy](https://bookkeeper.apache.org/docs/latest/api/javadoc/org/apache/bookkeeper/client/EnsemblePlacementPolicy.html) and it is a way to feed rack information which is stored as JSON format in znode.
+
+You can set a bookie affinity group using one of the following methods.
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Admin CLI-->
+
+```
+pulsar-admin namespaces set-bookie-affinity-group options
+```
+
+For more information about the command and descriptions, see [here](http://pulsar.apache.org/tools/pulsar-admin/2.8.0-SNAPSHOT/#-em-set-bookie-affinity-group-em-).
```

Comment:
we should not have a pointer to a specific version `2.8.0-SNAPSHOT`, probably for links to the Pulsar website itself we can use simply reletive references

Reply:
@eolivelli thanks for your suggestion. While the Pulsar website itself [(Pulsar Admin CLI doc)](https://pulsar.apache.org/docs/en/next/pulsar-admin/#policies) does not contain this command and the information on the "Pulsar Admin CLI" page is far behind from [pulsar-admin](http://pulsar.apache.org/tools/pulsar-admin/) where contains the lastest information. 
Seems that there was an issue (create automatic workflow) to sync contents from [pulsar-admin](http://pulsar.apache.org/tools/pulsar-admin/) to the Pulsar website, but no more updates for a long time. I think right now the best choice is to point users to get the latest information and we can update the link once we have a better optipon.